tests/strictEquals.test.js

```diff
@@ -0,0 +1,41 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEquals = require('../strictEquals')
+
+describe('Equals', () => {
+  describe('strictEquals', () => {
+    it('returns true when given equal values of the same type', () => {
+      const checkValueOne = 5
+      const checkValueTwo = 5
+
+      const checkResult = strictEquals(checkValueOne, checkValueTwo)
```

Comment:
Since this data is so simple, you could one line this

```JavaScript
const checkResult = strictEquals(5, 5)
```

Reply:
Thought that smelled funny.